apps/internal/base/internal/storage/storage.go

```diff
@@ -238,14 +236,10 @@ func (m *Manager) aadMetadata(ctx context.Context, authorityInfo authority.Info)
 	}
 
 	for _, metadataEntry := range discoveryResponse.Metadata {
-		metadataEntry.TenantDiscoveryEndpoint = discoveryResponse.TenantDiscoveryEndpoint
```

Comment:
Why the removal?

Reply:
Explained it in the PR description. Pasting here for reference : 
I have removed `TenantDiscoveryEndpoint` from the struct `instanceDiscoveryMetadata`. The reason being we have not actually used it from there. Also, this is how an InstanceDiscoveryResponse looks like : https://login.microsoftonline.com/common/discovery/instance?api-version=1.1&authorization_endpoint=https%3A%2F%2Flogin.microsoftonline.com%2F26039cce-489d-4002-8293-5b0c5134eacb%2Foauth2%2Fv2.0%2Fauthorize 

The metadata is only supposed to have `preferred_network`, `preferred_cache` and `aliases`.